bigframes/bigquery/__init__.py

```diff
@@ -272,6 +272,44 @@ def json_extract_array(
     return series._apply_unary_op(ops.JSONExtractArray(json_path=json_path))
 
 
+# Approximate aggrgate functions defined from
+# https://cloud.google.com/bigquery/docs/reference/standard-sql/approximate_aggregate_functions
+
+
+def approx_top_count(
+    series: series.Series,
+    number: int,
+) -> series.Series:
+    """Returns the approximate top elements of `expression` as an array of STRUCTs.
+    The number parameter specifies the number of elements returned.
+
+    Each `STRUCT` contains two fields. The first field (named `value`) contains an input
+    value. The second field (named `count`) contains an `INT64` specifying the number
+    of times the value was returned.
+
+    Returns `NULL` if there are zero input rows.
+
+    **Examples:**
+        >>> import bigframes.pandas as bpd
```

Comment:
I think we need newline after **Examples:** to render it nicely

Reply:
Good catches! Fixed.